main.go

```diff
@@ -59,16 +61,29 @@ func run(ctx context.Context) error {
 	svc := service.New()
 	svc.IntiateServiceList(cfg, svcList)
 
+	navCache, err := cacheHelper.Init(ctx, cacheHelper.Config{
```

Comment:
Maybe this should go in the service initialization? (service package instead of main) but there is also some code to initialise clients later in this file anyway.

Reply:
Also, I see there is a single service.Run func, so it would be a bigger change, maybe let's leave it here.